.github/workflows/ui-tests.yml

```
@@ -77,18 +78,22 @@ jobs:
-          DEVICE_ID=$(xcrun simctl list devices available | grep "$DEVICE_NAME" | grep "$OS_VERSION" | head -1 | grep -oE '\[.*\]' | tr -d '[]')
-          if [ ! -z "$DEVICE_ID" ]; then
-            echo "Booting simulator: $DEVICE_NAME ($OS_VERSION) - $DEVICE_ID"
-            xcrun simctl boot "$DEVICE_ID" || true
-            echo "Waiting for simulator to fully boot..."
-            xcrun simctl bootstatus "$DEVICE_ID" -b
-            echo "Simulator is ready!"
+          # Check if this is a simulator destination
+          if [[ '${{ matrix.destination }}' == *"Simulator"* ]]; then
+            # Extract device details from destination
+            DEVICE_NAME=$(echo '${{ matrix.destination }}' | sed -n 's/.*name=\([^,)]*\).*/\1/p')
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)